### PR TITLE
Add travis config and auto deployment

### PR DESCRIPTION
**Pull request by slomo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: node_js
+install:
+- npm install
+- npm run bower:install
```
Comment (cbotsikas) [inline .travis.yml]: why did you add this? can't use "bower install" command here?

Comment (slomo) [inline .travis.yml]: It requires me to run `node_modules/.bin/bower install`. This is in my opinion less pleasing to the eye, making more assumptions about paths and less npm-style.

Comment (cbotsikas) [inline .travis.yml]: nope, you don't need to. see https://github.com/policycompass/policycompass-frontend/pull/464

Comment (cbotsikas) [inline .travis.yml]: the reason why it's working is because the nodeJS travis nodes has "./node_modules/.bin" in $PATH already

Comment (cbotsikas) [inline .travis.yml]: We can solve the problem you have on a separate commit. In fact i'm already working with a grunt solution, so i can address that too. 
In my environment i have bower globally installed (npm install -g bower).

**Comment by cbotsikas** [comment]
I'm closing this in favor of https://github.com/policycompass/policycompass-frontend/pull/464
